walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,86 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+
+  private final int TOTAL_DAY_MINUTES = 60 * 24;
+
```

Comment:
nit: could add a javadoc comment here as well

Reply:
Sure, thanks for the recommendation.